cmake/CMakeLists.txt

```diff
@@ -1397,7 +1396,6 @@ if (onnxruntime_USE_NUPHAR_TVM)
     endif()
   endif()
 
-  include(onnxruntime_codegen.cmake)
```

Comment:
I need to put this line at a later location. Because here we haven't setup the necessary compile flags for ORT yet.